homepage/forms.py

```diff
@@ -7,8 +7,11 @@
 
 class ReviewForm(ModelForm):
     def __init__(self, *args, **kwargs):
+        # self.request = kwargs.pop('request')

```

Comment:
do we need this line?
if no, please remove it.

Reply:
Removed